documentation/.eslintrc

```diff
@@ -0,0 +1,101 @@
+env:
```

Comment:
✂️ ?

Reply:
I think this has something to do with `eslint` that says it's in the browser, not on a node server?

Reply:
Yeah it looks like it provides some global variables https://eslint.org/docs/user-guide/configuring#specifying-environments